test_scripts/Defects/5_0/2481_ChoiceID_is_missing_in_PerformInteraction_response.lua

```diff
@@ -0,0 +1,144 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_core/issues/2481
+-- Description:
+-- 1) ChoiceID is missing in PerformInteraction response
+-- Precondition:
```

Comment:
@TMyza Please modify precondition and steps to reproduce for them become applicable to automated script.